src/js/models/skybox.js

```diff
@@ -1,25 +1,51 @@
 import colors from '../constants/colors';
 import Sphere from './sphere'
+import sunsetImage from '../../assets/maps/sunset.jpg'
 
 export default class Skybox extends Sphere {
     color() {
         return colors.skyBlue;
     }
 
+    initTexture(filePath) {
+        this.texture = gl.createTexture();
+        this.texture.image = new Image();
+        
+        let texture = this.texture;
+		this.texture.image.onload = function () {
```

Comment:
pasale 1 linter plz jajaja

https://stackoverflow.com/questions/29973357/how-do-you-format-code-in-visual-studio-code-vscode